### PR TITLE
Providers: domain -> issuer

### PR DESCRIPTION
**Pull request by Thisen**
## Changes 💡

Update docs for providers, that previously used `domain` to `issuer`, as `domain` has been removed.

## Affected issues 🎟

Haven't found any.

## Screenshot (If Applicable) 📷

N/A

### CODE REVIEW

File: docs/providers/identity-server4.md [hunk 1]
```diff
@@ -48,11 +47,10 @@ providers: [
   IdentityServer4Provider({
     id: "demo-identity-server",
     name: "Demo IdentityServer4",
-    scope: "openid profile email api offline_access",
-    domain:  "demo.identityserver.io",
+    authorization: { params: { scope: "openid profile email api offline_access" } },
+    issuer:  "https://demo.identityserver.io/",
```
Comment (balazsorban44) [inline docs/providers/identity-server4.md:51]: I am fairly sure I tested this locally, and I don't think I needed any of these changes. But I will check again

**Review by balazsorban44 — COMMENTED** [review]
Thanks, I think this PR is alright, but I'll just check the IDS4 provider before approving.

**Comment by Thisen** [comment]
> Thanks, I think this PR is alright, but I'll just check the IDS4 provider before approving.

@balazsorban44 The reasoning beind it, is that when I migrated an application to v4, I got the following error: `only valid absolute URLs can be requested`
Then I changed it to an absolute URL and it worked.

File: docs/providers/identity-server4.md (hunk 1 above)
Comment (balazsorban44) [reply docs/providers/identity-server4.md:51]: You were right here! :+1:

**Review by balazsorban44 — APPROVED** [review]

**Pull request merged**